src/main/java/org/mbari/m3/dive/visualizer/DiveAnnotationService.java

```diff
@@ -0,0 +1,161 @@
+package org.mbari.m3.dive.visualizer;
+
+import io.helidon.webserver.Routing;
+import io.helidon.webserver.ServerRequest;
+import io.helidon.webserver.ServerResponse;
+import io.helidon.webserver.Service;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpHeaders;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import java.util.Collection;
+import java.util.Map;
+
+import org.mbari.expd.Dive;
+import org.mbari.expd.DiveDAO;
+import org.mbari.expd.jdbc.DiveDAOImpl;
+import org.json.simple.JSONObject;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import io.helidon.webserver.Routing;
+
+public class DiveAnnotationService implements Service {
+
+    @Override // this is called everytime this path is accessed
+    public void update(Routing.Rules rules) {
+        rules.get("/{rov}", (req, res) -> {
+            try {
+                getRovDiveAnnotations(req, res);
+            } catch (IOException | InterruptedException e) {
+                // TODO Auto-generated catch block
+                e.printStackTrace();
+            }
+        });
+    }
+    
+
+    /**
+     * Returns a list of dives for the given ROV.
+     * 
+     * @param request
+     * @param response
+     * @throws InterruptedException
+     * @throws IOException
+     */
+    private void getRovDiveAnnotations(ServerRequest request, ServerResponse response) throws IOException, InterruptedException {
+        String rovname = "";
+        String rovgiven = request.path().param("rov");
```

Comment:
```suggestion
        String rovGiven = request.path().param("rov");
```
Stick with camel case convention.